webapp/package.json

```diff
@@ -17,34 +17,34 @@
     "npm": ">=6.4.1"
   },
   "dependencies": {
-    "@angular/animations": "^12.2.1",
-    "@angular/common": "^12.2.1",
-    "@angular/compiler": "^12.2.1",
-    "@angular/core": "^12.2.1",
-    "@angular/forms": "^12.2.1",
-    "@angular/platform-browser": "^12.2.1",
-    "@angular/platform-browser-dynamic": "^12.2.1",
-    "@angular/router": "^12.2.1",
+    "@angular/animations": "^12.2.16",
+    "@angular/common": "^12.2.16",
+    "@angular/compiler": "^12.2.16",
+    "@angular/core": "^12.2.16",
+    "@angular/forms": "^12.2.16",
+    "@angular/platform-browser": "^12.2.16",
+    "@angular/platform-browser-dynamic": "^12.2.16",
+    "@angular/router": "^12.2.16",
     "@ngrx/effects": "^11.1.1",
     "@ngrx/store": "^11.1.1",
     "@ngx-translate/core": "^13.0.0",
     "bikram-sambat": "^1.7.0",
     "bikram-sambat-bootstrap": "^1.5.0",
     "bootstrap": "^3.4.1",
     "bootstrap-daterangepicker": "^2.1.30",
-    "enketo-core": "4.41.6",
+    "enketo-core": "4.60.6",
```

Comment:
Maybe we can consider leaving Enketo in 4.41.6, I see in their [change logs](https://github.com/enketo/enketo-core/blob/master/CHANGELOG.md) some warnings and fixes that not sure if it affects how people is used to use the forms. Maybe leaving all that testing effort once we release the upgrade in CHT-Core v4.0.0 with Josh work.

Reply:
agreed! reverted in dfc5859e0